### ISSUE TITLE
[v1.9] CI: k8s-upstream (test-upstream-k8s) add_vagrant_box timeout

### ISSUE THREAD

**Issue by kaworu** [cilium/cilium#19079]
## CI failure

**k8s-upstream (test-upstream-k8s)** failed thrice on [this v1.9 backport PR](https://github.com/cilium/cilium/pull/18986) timing out on vagrant.

- Jenkins build link: https://jenkins.cilium.io/job/Cilium-PR-K8s-Upstream/3518/
- full CI failure output: [Cilium-PR-K8s-Upstream-3518.log](https://github.com/cilium/cilium/files/8205609/Cilium-PR-K8s-Upstream-3518.log)

<details>
<summary>CI failure output (truncated)</summary>

```
00:28:34.434  + /usr/local/bin/add_vagrant_box /home/jenkins/workspace/Cilium-PR-K8s-Upstream/src/github.com/cilium/cilium/vagrant_box_defaults.rb
00:28:36.643    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
00:28:36.643                                   Dload  Upload   Total   Spent    Left  Speed
00:28:36.643  
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
00:28:36.643  curl: (22) The requested URL returned error: 404 Not Found
00:28:36.643  adding box from cache
00:28:36.643    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
00:28:36.643                                   Dload  Upload   Total   Spent    Left  Speed
00:28:36.963  
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   321  100   321    0     0   2309      0 --:--:-- --:--:-- --:--:--  2309
00:28:36.963    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
00:28:36.963                                   Dload  Upload   Total   Spent    Left  Speed
00:34:24.841  
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 8674M    0 6661k    0     0  6185k      0  0:23:56  0:00:01  0:23:55 6185k
  0 8674M    0 8308k    0     0  3988k      0  0:37:06  0:00:02  0:37:04 3986k

  7 8674M    7  620M    0     0  10.3M      0  0:13:59  0:01:00  0:12:59 18.6M
  7 8674M    7  640M    0     0  10.4M      0  0:13:47  0:01:01  0:12:46 18.9M
  7 8674M    7  660M    0     0  10.6M      0  0:13:35  0:01:02  0:12:33 19.3M
…
 99 8674M   99 8588M    0     0  24.9M      0  0:05:47  0:05:44  0:00:03 20.7M
 99 8674M   99 8610M    0     0  24.9M      0  0:05:47  0:05:45  0:00:02 21.0M
 99 8674M   99 8632M    0     0  24.9M      0  0:05:47  0:05:46  0:00:01 21.3M
 99 8674M   99 8654M    0     0  24.9M      0  0:05:47  0:05:47 --:--:-- 21.7M
100 8674M  100 8674M    0     0  24.9M      0  0:05:47  0:05:47 --:--:-- 22.0M
00:34:28.558  ==> box: Loading metadata for box 'metadata.json'
00:34:28.558      box: URL: file:///home/jenkins/workspace/Cilium-PR-K8s-Upstream/metadata.json
00:34:28.558  ==> box: Adding box 'cilium/ubuntu' (v202) for provider: virtualbox
00:34:28.558      box: Downloading: package.box
00:34:47.225  
[KProgress: 0% (Rate: 0/s, Estimated time remaining: --:--:--)
[KProgress: 4% (Rate: 459M/s, Estimated time remaining: 0:00:18)
[KProgress: 9% (Rate: 463M/s, Estimated time remaining: 0:00:17)
[KProgress: 15% (Rate: 462M/s, Estimated time remaining: 0:00:16)
[KProgress: 20% (Rate: 463M/s, Estimated time remaining: 0:00:15)
[KProgress: 25% (Rate: 463M/s, Estimated time remaining: 0:00:14)
[KProgress: 31% (Rate: 462M/s, Estimated time remaining: 0:00:13)
[KProgress: 36% (Rate: 461M/s, Estimated time remaining: 0:00:12)
[KProgress: 41% (Rate: 458M/s, Estimated time remaining: 0:00:11)
[KProgress: 46% (Rate: 454M/s, Estimated time remaining: 0:00:10)
[KProgress: 51% (Rate: 451M/s, Estimated time remaining: 0:00:09)
[KProgress: 56% (Rate: 447M/s, Estimated time remaining: 0:00:09)
[KProgress: 61% (Rate: 439M/s, Estimated time remaining: 0:00:08)
[KProgress: 66% (Rate: 434M/s, Estimated time remaining: 0:00:07)
[KProgress: 72% (Rate: 438M/s, Estimated time remaining: 0:00:06)
[KProgress: 76% (Rate: 432M/s, Estimated time remaining: 0:00:05)
[KProgress: 82% (Rate: 434M/s, Estimated time remaining: 0:00:04)
[KProgress: 87% (Rate: 440M/s, Estimated time remaining: 0:00:03)
[KProgress: 92% (Rate: 450M/s, Estimated time remaining: 0:00:02)
[KProgress: 97% (Rate: 449M/s, Estimated time remaining: 0:00:01)
[K    box: Calculating and comparing box checksum...
00:38:55.852  ==> box: Successfully added box 'cilium/ubuntu' (v202) for 'virtualbox'!
00:38:58.061    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
00:38:58.061                                   Dload  Upload   Total   Spent    Left  Speed
00:38:58.380  
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
00:38:58.380  curl: (22) The requested URL returned error: 404 Not Found
00:38:58.380  adding box from cache
00:38:58.380    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
00:38:58.380                                   Dload  Upload   Total   Spent    Left  Speed
00:38:58.699  
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   325  100   325    0     0   2338      0 --:--:-- --:--:-- --:--:--  2338
00:38:58.699    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
00:38:58.699                                   Dload  Upload   Total   Spent    Left  Speed
00:47:07.227  
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 13.0G    0 14204    0     0    99k      0 38:30:32 --:--:-- 38:30:32   98k
  0 13.0G    0 4981k    0     0  4365k      0  0:52:26  0:00:01  0:52:25 4362k
  0 13.0G    0 6344k    0     0  2957k      0  1:17:23  0:00:02  1:17:21 2956k
  0 13.0G    0 8020k    0     0  2546k      0  1:29:52  0:00:03  1:29:49 2546k
…
 99 13.0G   99 12.9G    0     0  27.3M      0  0:08:10  0:08:06  0:00:04 51.2M
 99 13.0G   99 13.0G    0     0  27.4M      0  0:08:09  0:08:07  0:00:02 51.4M
 99 13.0G   99 13.0G    0     0  27.4M      0  0:08:08  0:08:08 --:--:-- 51.5M
100 13.0G  100 13.0G    0     0  27.4M      0  0:08:08  0:08:08 --:--:-- 51.6M
00:47:10.941  ==> box: Loading metadata for box 'metadata.json'
00:47:10.942      box: URL: file:///home/jenkins/workspace/Cilium-PR-K8s-Upstream/metadata.json
00:47:10.942  ==> box: Adding box 'cilium/ubuntu-next' (v97) for provider: virtualbox
00:47:11.261      box: Downloading: package.box
00:47:46.936  
[KProgress: 0% (Rate: 0/s, Estimated time remaining: --:--:--)
[KProgress: 2% (Rate: 462M/s, Estimated time remaining: 0:00:29)
[KProgress: 6% (Rate: 464M/s, Estimated time remaining: 0:00:27)
[KProgress: 9% (Rate: 460M/s, Estimated time remaining: 0:00:27)
[...]
[KProgress: 90% (Rate: 290M/s, Estimated time remaining: 0:00:04)
[KProgress: 93% (Rate: 284M/s, Estimated time remaining: 0:00:03)
[KProgress: 96% (Rate: 389M/s, Estimated time remaining: 0:00:02)
[KProgress: 99% (Rate: 412M/s, Estimated time remaining: --:--:--)
[K    box: Calculating and comparing box checksum...
00:48:33.930  Sending interrupt signal to process
00:48:44.555  Terminated
00:48:44.602  script returned exit code 143
```
</details>

**Comment by nbusseneau** [comment]
Will be fixed by #18986 (see https://github.com/cilium/cilium/pull/18707#issuecomment-1061839149 for details).

**Comment by nbusseneau** [comment]
Was fixed by https://github.com/cilium/cilium/pull/18986.
